ledger/ledger.go

```diff
@@ -805,22 +803,22 @@ func (l *Ledger) Validate(ctx context.Context, blk bookkeeping.Block, executionP
 	return &vb, nil
 }
 
-// CompactCertParams computes the parameters for building or verifying
-// a compact cert for block hdr, using voters from block votersHdr.
-func CompactCertParams(votersHdr bookkeeping.BlockHeader, hdr bookkeeping.BlockHeader) (res compactcert.Params, err error) {
-	return internal.CompactCertParams(votersHdr, hdr)
+// GetProvenWeight computes the provenWeight for building or verifying
+// a state proof for block hdr, using voters from block votersHdr.
+func GetProvenWeight(votersHdr bookkeeping.BlockHeader, hdr bookkeeping.BlockHeader) (uint64, error) {
+	return internal.GetProvenWeight(votersHdr, hdr)
```

Comment:
Hopefully we can get rid of useless wrappers like this when we rename internal to a different package name as in https://github.com/algorand/go-algorand/pull/4272